week-1/F-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var message = "Hello my name is";
```

Comment:
Nice Brat .Consider putting space before closing string on line 2

Reply:
Thanks Madiha! Have updated this now.